### DISCUSSION TITLE
Using mix input in media_data

### DISCUSSION THREAD

**Discussion by mahewashabdi** [google/lightweight_mmm#19] [General]
Hi 

I am trying to create a MMM model. I have few questions regarding the input and prices ( in optimized budget allocation).

1) Is possible to use mix of impressions and clicks in the media_data input for the model (or only clicks or only impressions)?
2) Also can I use the prices(for optimzed budget allocation) as mix of cpc and cpm depending upon how marketing is charged ( As per the documentation I shouldn't be using the prices if media_data is in money unit like impressions or clicks) but then the sum of solution is not equal to the budget) ?
![image](https://user-images.githubusercontent.com/63654651/173624683-cad3e1d5-3f8e-4a07-9816-b037f18e1dca.png)

Many Thanks

**Comment by pabloduque0** [comment]
Hi @mahewashabdi !

Let me try to answer them:

1. You can have a mix of impressions for some channels and clicks for others. But you cannot mix both in the same channel.
2. Yes, I think you should be able to use a CPC for the channels you used clicks and CPM / 1000 for the ones you used impressions.

Hope it helps!
